test/unit_test/automl_test.cc

```diff
@@ -19,22 +19,6 @@ using simulator::callback_map;
 using simulator::cb_sim;
 using namespace VW::reductions::automl;
 
-#define TEST_SCORE_CLEARED(aml)                    \
```

Comment:
is this temp

Reply:
this test isn't valid anymore since the scores aren't cleared on a champ change. The old and incoming champ both stay live